src/NUnitEngine/nunit.engine.api/TestSelectionParserException.cs

```diff
@@ -15,18 +15,24 @@ public class TestSelectionParserException : Exception
         /// <summary>
         /// Construct with a message
         /// </summary>
-        public TestSelectionParserException(string message) : base(message) { }
+        public TestSelectionParserException(string message) : base(message)
+        {
+        }
 
         /// <summary>
         /// Construct with a message and inner exception
         /// </summary>
         /// <param name="message"></param>
         /// <param name="innerException"></param>
-        public TestSelectionParserException(string message, Exception innerException) : base(message, innerException) { }
+        public TestSelectionParserException(string message, Exception innerException) : base(message, innerException)
+        {
+        }
```

Comment:
I see a number of these. Depending on what happens when I try to put the empty braces on the same line, I may get annoyed at StyleCop for not being smart enough to see that there is nothing in the braces so don't worry. :-)

But let's give it a go anyway.